docs/pages/repo/docs/installing.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Installing Turborepo
+description: Learn how to install Turborepo for use with your repository
+---
+
+import Callout from "../../../components/Callout";
+import { Tabs, Tab } from '../../../components/Tabs'
+
+# Install Turborepo
+
+`turbo` works with [Yarn](https://classic.yarnpkg.com/lang/en/), [npm](https://www.npmjs.com/), and [pnpm](https://pnpm.io/) on the following operating systems:
+
+- macOS darwin 64-bit (Intel), ARM 64-bit (Apple Silicon)
+- Linux 64-bit, ARM 64-bit
+- Windows 64-bit, ARM 64-bit
+
+<Callout>
+  Note: Linux builds of `turbo` link against `glibc`. For Alpine Docker environments, you will need to ensure libc6-compat is installed as well, via `RUN apk add --no-cache libc6-compat`
```

Comment:
Do we still link against glibc? I thought we switched to musl?

Reply:
You do still need `libc6-compat` in alpine.

Reply:
I cut and paste'd this, and it's a little bit of a simplification. The real answer is something like:
"The Go executable portion of Turborepo expects to dynamically link against glibc. This means that Alpine Linux users will need to ensure the `libc6-compat` package is installed. To install `libc6-compat` when creating a Docker image based on Alpine Linux, include the line `RUN apk add --no-cache libc6-compat` in your Dockerfile"

FWIW I think that's a bit too much detail, but I'm open to suggestions on a rewording here if anyone has some good ideas.